DockerfileOp

```diff
@@ -6,24 +6,30 @@ LABEL org.opencontainers.image.licenses="MIT OR Apache-2.0"
 
 RUN apt-get update && apt-get -y upgrade && apt-get install -y libclang-dev pkg-config
 
+# Builds a cargo-chef plan
 FROM chef AS planner
-COPY . .
+COPY --exclude=.git --exclude=dist . .
```

Comment:
we need to keep git for things like sha etc, so please undo this, this is unrelated to the issue anyway